Mainnet/StandardToken/StandardToken/StandardToken.cs

```diff
@@ -13,12 +13,13 @@ public class StandardToken : SmartContract, IStandardToken
     /// <param name="totalSupply">The total token supply.</param>
     /// <param name="name">The name of the token.</param>
     /// <param name="symbol">The symbol used to identify the token.</param>
-    public StandardToken(ISmartContractState smartContractState, ulong totalSupply, string name, string symbol)
+    public StandardToken(ISmartContractState smartContractState, UInt256 totalSupply, string name, string symbol, uint decimals)
         : base(smartContractState)
     {
         this.TotalSupply = totalSupply;
         this.Name = name;
         this.Symbol = symbol;
+        this.Decimals = decimals;
```

Comment:
Decimals type should to be `byte` rather than `uint` like in orginal

Reply:
This is using `uint` to match Mr TPain's PR 17.

> like in original
?

Reply:
I mean original implementations from solidity and other projects prefer type as byte which is better.

Reply:
I had `uint` but @YakupIpek made the suggestion of using `byte` which would be better. Oversight on my part for using uint.